### ISSUE TITLE
build

### ISSUE THREAD

**Issue by sberinde** [wrandelshofer/CubeTwister#3]
Hi,
Can you provide at some point a pure java build with latest features?
My latest build is from 2014 (v. 2.0.155).
Thanks.

**Comment by wrandelshofer** [comment]
I was thinking about creating native builds with the new jpackage tool in Java 14. 
How would you use a pure java build? Do you use CubeTwister as a software library?

**Comment by sberinde** [comment]
No, I mean a .jar file without installer. I used CubeTwister over the years to test stickers designs.
Also, I noticed your javascript library is working very nice now. Hope AutoPlay will be implemented soon!

**Comment by SimonW-dev** [comment]
Latest build is still from 2014. Have you given any thought to this? I would also really appreciate an updated pure java release. 

I'm planning to use it as a visualizer for my professors deep learning trainer on rubiks cubes.
